app/views/npq_separation/admin/glossary/index.html.erb

```diff
@@ -0,0 +1,362 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <h1 class="govuk-heading-l">Glossary</h1>
+
+    <p class="govuk-body">The NPQ glossary explains the terms used by DfE teams working on national professional qualifications.</p>
+
+    <ul class="govuk-list" style="display: flex; flex-wrap: wrap;">
+      <% ("A".."Z").each do |letter| %>
+        <li class="govuk-!-padding-right-3 govuk-!-padding-bottom-2">
+          <a class="govuk-link govuk-body-m govuk-!-padding-2" href="#<%= letter %>"><%= letter %></a>
+        </li>
+      <% end %>
+    </ul>
+
+    <h2 id="A" class="govuk-heading-l">A</h2>
+    <h3 class="govuk-heading-s">Application</h3>
+    <p class="govuk-body">The user applies to join a course run by a training provider. This happens when they register with the Department for Education (DfE).</p>
+
+    <h3 class="govuk-heading-s">Application ID</h3>
+    <p class="govuk-body">Each application has a unique ID. You can use this ID to find a specific application. Application ID example: 0594fe7b-fcd2-4f3f-9e8d-17ab5538a2b9.</p>
+
+    <h3 class="govuk-heading-s">Application programming interface (API)</h3>
+    <p class="govuk-body">A set of tools and protocols that allow different software systems to securely connect with ours to:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>collect new applications</li>
+      <li>accept or reject applications</li>
+      <li>update application, participant and declaration details</li>
+      <li>submit declarations and outcomes</li>
+    </ul>
+
+    <h3 class="govuk-heading-s">Application status</h3>
+    <p class="govuk-body">The status shows if a user is currently taking part in an NPQ course. The status can be:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>'Active' – they're currently doing the course</li>
```

Comment:
```suggestion
      <li>‘Active’ – they‘re currently doing the course</li>
```
do we want to use smart/curly quotes throughout, for consistency with the other quotes on the site.

Reply:
Good spot! Yes